setup.py

```diff
@@ -34,7 +34,7 @@
     # Until this issue is closed
     # https://github.com/googleapis/google-cloud-python/issues/10566
     "google-api-core[grpc] >= 1.31.5, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
```

Comment:
Likewise, it's possible we can remove the `[grpc]` extra from here now that `google.cloud.bigquery_v2` has no connection to `google.cloud.bigquery`.

Reply:
Tracked in https://github.com/googleapis/python-bigquery/issues/1431